.travis.yml

```diff
@@ -13,8 +13,9 @@ matrix:
     - php: 7.0
     - php: 7.1
     - php: 7.1
-      env: DEPENDENCIES='dev'
+      env: COMPOSER_FLAGS='--prefer-lowest'
```

Comment:
you replace the wrong config here. This is the replacement for `DEPENDENCIES='low'`, not for `DEPENDENCIES='dev'`